model/attributes.yaml

```diff
@@ -1,187 +1,208 @@
-      - id: policy.name
+      - id: policy.engine.version
         type: string
         stability: development
         brief: >
-          The human-readable name of the policy
+          Version of the policy engine.
+        examples: [ "v3.14.0", "v0.45.0", "v1.2.3", "v2.0.1" ]
         requirement_level: recommended
-        examples: ["S3 Bucket Check"]
-      - id: policy.source
+      - id: policy.rule.id
         type: string
         stability: development
         brief: >
-          The identifier for the source of the policy audit log.
-        note:
-          This should identify the policy engine or assessment tool.
-        examples:
-          ["conforma", "opa-gatekeeper", "kyverno"]
+          Unique identifier for the policy rule being evaluated or enforced.
+        examples: [ "deny-root-user", "require-encryption", "check-labels" ]
         requirement_level: required
-      - id: policy.evaluation.status
+      - id: policy.rule.name
+        type: string
+        stability: development
+        brief: >
+          Human-readable name of the policy rule.
+        examples: [ "Deny Root User", "Require Encryption", "Check Resource Labels" ]
+        requirement_level: opt_in
+      - id: policy.rule.uri
+        type: string
+        stability: development
+        brief: >
+          Source control URL and version of the policy-as-code file for auditability.
+        examples:
+          ["github.com/org/policy-repo/b8a7c2e", "gitlab.com/company/policies@v1.2.3"]
+        requirement_level: recommended
+      - id: policy.evaluation.result
         type: string
         stability: development
         brief: >
-          The outcome of the policy evaluation (e.g., "deny", "allow").
-        examples: [ "pass", "fail", "error" ]
+          Result of the policy evaluation: Not Run, Passed, Failed, Needs Review, Not Applicable, or Unknown.
+        examples: [ "Not Run", "Passed", "Failed", "Needs Review", "Not Applicable", "Unknown" ]
         requirement_level: required
-      - id: policy.status.detail
+      - id: policy.evaluation.message
         type: string
         stability: development
         brief: >
-          Add contextual details around the policy status.
+          Additional context about the policy evaluation result.
         requirement_level: opt_in
         examples: ["The policy evaluation failed due to a missing attribute."]
-      - id: policy.evaluation.subject.type
+      - id: policy.target.id
         type: string
         stability: development
         brief: >
-          The subject type or resource the policy was applied to.
-        requirement_level: recommended
-        examples: ["k8s.pod", "k8s.service", "gcp.compute.instance"]
-      - id: policy.evaluation.subject.id
+          Unique identifier for the resource or entity being evaluated or enforced against.
+        examples: [ "deployment-123", "resource-456", "user-789" ]
+        requirement_level: opt_in
```

Comment:
I believe the `policy.target.id` should be required. We have the `engine.name` and `engine.version` that was used to process the `rule.id` and generate the `evaluation.result` but this result would not bring much value without the target information IMO. Is there any case we can't define a target?